manifests/o/Oracle/JDK/17/17.0.8/Oracle.JDK.17.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Oracle.JDK.17
+PackageVersion: 17.0.8
```

```suggestion
PackageVersion: 17.0.8.0
```
Must match Control Panel / Apps&Features